packages/ui/src/forum/components/threads/ThreadItem.tsx

```diff
@@ -7,56 +7,61 @@ import { Arrow } from '@/common/components/icons'
 import { ColumnGapBlock } from '@/common/components/page/PageContent'
 import { Label } from '@/common/components/typography'
 import { BorderRad, Colors, Transitions } from '@/common/constants'
+import { ForumThread } from '@/forum/types'
 
 import { ThreadItemContent } from './ThreadItemContent'
 import { ThreadsLayoutSpacing } from './ThreadsLayout'
 
 export interface ThreadItemProps {
-  label: string
-  count?: number
+  categoryLabel: string
+  categoryCount?: number
+  thread: ForumThread
+  withButtons?: boolean
```

Comment:
I think that it makes API more convenient when using false as the default value for optional parameters.

Using default `true` requires more typing:

```typescript
interface Props {
  withButtons?: boolean // default true
}

// Hide buttons:
<ThreadItem withButtons={false} />

// Show buttons:
<ThreadItem />
```

Using default `false` tends to be more fluent:

```typescript
interface Props {
  withButtons?: boolean // default false
}

// Hide buttons:
<ThreadItem />

// Show buttons:
<ThreadItem withButtons />
```

Using this for React components also makes them similar to HTML attributes:

```html
<button disabled />
```